toolkits/linear/evals/eval_teams.py

```diff
@@ -0,0 +1,156 @@
+"""
+Evaluation suite for Linear get_teams tool.
+"""
+
+from arcade_evals import (
+    BinaryCritic,
+    EvalRubric,
+    EvalSuite,
+    ExpectedToolCall,
+    SimilarityCritic,
+    tool_eval,
+)
+from arcade_tdk import ToolCatalog
+
+import arcade_linear
+from arcade_linear.tools.teams import get_teams
+
+# Evaluation rubric
+rubric = EvalRubric(
+    fail_threshold=0.85,
+    warn_threshold=0.95,
+)
+
+catalog = ToolCatalog()
+catalog.add_module(arcade_linear)
+
+
+@tool_eval()
+def teams_eval_suite() -> EvalSuite:
+    """Comprehensive evaluation suite for get_teams tool"""
+    suite = EvalSuite(
+        name="Teams Management Evaluation",
+        system_message=(
+            "You are an AI assistant with access to Linear tools. "
+            "Use them to help the user manage Linear teams and organizational structure."
+        ),
+        catalog=catalog,
+        rubric=rubric,
+    )
+
+    # Eval Prompt: Show me all teams in our workspace
+    suite.add_case(
+        name="Get all teams in workspace",
+        user_message="Show me all teams in our workspace",
+        expected_tool_calls=[
+            ExpectedToolCall(
+                func=get_teams,
+                args={},
+            ),
+        ],
+        critics=[],  # No specific args expected
+    )
+
+    # Eval Prompt: Which teams were created in the last month?
+    suite.add_case(
+        name="Find recently created teams",
+        user_message="Which teams were created in the last month?",
+        expected_tool_calls=[
+            ExpectedToolCall(
+                func=get_teams,
+                args={
+                    "created_after": "last month",
+                },
+            ),
+        ],
+        critics=[
+            SimilarityCritic(critic_field="created_after", weight=1.0),
```

Comment:
i think this should be BinaryCritic. For example, "lastmonth" would cause the tool to fail, but the eval would give it a great score. Applies to the cases below too